### ISSUE TITLE
Java installation description in Readme.md

### ISSUE THREAD

**Issue by lbthomsen** [subutai-io/base#2153]
The approach is slightly messy.  Cleaner on Debian/Ubunto would be to use the tools for handling java packages.  Assuming jdk-8u152-linux-x64.tar.gz has been downloaded from Oracle, the following steps should work on both Debian and Ubuntu:

 sudo apt-get install java-common java-package

Then use make-jpkg to build a Debian file from the downloaded package:

 make-jpkg Downloads/jdk-8u152-linux-x64.tar.gz

This will result in a oracle-java8-jdk_8u152_amd64.deb which can simply be installed:

 sudo dpkg --install oracle-java8-jdk_8u152_amd64.deb

The java-common provides update-java-alternatives which can be used to switch jvm at any time.

**Comment by akarasulu** [comment]
Vagrant or Docker image to the rescue?

**Comment by lbthomsen** [comment]
No, no - this was merely a comment on the readme.  The way it is described in the readme will work, but it will confuse matters if a previous debian/ubuntu java is installed already (as in some links will point to different jvms).  The approach described above will ensure that it doesn't conflict with Debian's way of doing things.
